src/Umbraco.Examine.Lucene/Umbraco.Examine.Lucene.csproj

```diff
@@ -1,22 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-    <RootNamespace>Umbraco.Cms.Infrastructure.Examine</RootNamespace>
-    <Product>Umbraco CMS</Product>
-    <Title>Umbraco.Examine.Lucene</Title>
-    <!-- But we still need to have PackageId for the depdents to know the actual name of the package. -->
     <PackageId>Umbraco.Cms.Examine.Lucene</PackageId>
+    <Title>Umbraco CMS - Examine - Lucene</Title>
+    <Description>Adds Examine searching support using Lucene to Umbraco CMS.</Description>
+    <RootNamespace>Umbraco.Cms.Infrastructure.Examine</RootNamespace>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Examine" Version="3.0.1" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
-    <PackageReference Include="Umbraco.Code" Version="2.0.0">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
```

Comment:
Always a bit nervous when removing dependencies, but I guess these aren't needed? or do we get them from the core project ref even though they're marked private assets?

Reply:
Oh I see! That's really neat 😄